src/transpilation/bundle.ts

```diff
@@ -77,11 +77,11 @@ export function getBundleResult(program: ts.Program, files: ProcessedFile[]): [t
     ];
 }
 
-function moduleSourceNode({ code, sourceMapNode }: ProcessedFile, modulePath: string): SourceNode {
+function moduleSourceNode({ code }: ProcessedFile, modulePath: string): SourceNode {
     const tableEntryHead = `[${modulePath}] = function() `;
     const tableEntryTail = " end,\n";
 
-    return joinSourceChunks([tableEntryHead, sourceMapNode ?? code, tableEntryTail]);
```

Comment:
Seems like you're just throwing away the source map information of the file? Did you test this? Seems to me like this would just break the source map completely.

Reply:
The code already contains source map information. I'll figure it out and write tests tomorrow.

Reply:
I've changed the source map tracebacks behavior for bundles. The details are below